syntax="proto3";

message CreateProjectRequest{
    string ProjectId = 1;
    string ProjectName = 2;
}
message CreateProjectResponse{}


service Catalog {
    rpc CreateProject(CreateProjectRequest) returns (CreateProjectResponse){}

}

message CreateDataset {
    string dataset_id = 1;
    string project_id = 2;
}






